docs/programmatic-usage.md

```diff
@@ -15,12 +15,12 @@ This will download the package `requests`, scan it, then cleanup the temporary d
 
 ## API
 
-### Class `PackageScanner`
+### Class `PypiPackageScanner`
```

Comment:
What about NPM ones? It may be time to try and autogenerate the docs?

No matter what we choose we should be consistent, so either we document both implementations or none

Reply:
Will do